docs/book/src/migration/guide.md

```diff
@@ -0,0 +1,192 @@
+# Migration Guide
+
+This document describes how to migrate a project from kubebuilder v1 to v2.
+
+We suggest you read [this doc](./v1vsv2.md) that compares the notable differences between v1 and v2.
+
+The recommended way of migration is to create a new v2 project and copy the business logic code.
+
+Please ensure you have followed the [installation guide](../quick-start.md#installation) to install the required
```

Comment:
use absolute paths when referencing unrelated parent elements -- it makes it more obvious here